unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/ServerBuilder/RPCServerBuilder.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using rpc_csharp;
+using rpc_csharp.transport;
+using RPC.Services;
+using RPC.Transports;
+using Environment = DCL.Environment;
+
+namespace RPC
+{
+    public static class RPCServerBuilder
+    {
+        public static void BuildDefaultServer()
+        {
+            RPCGlobalContext.context.crdtContext.messageQueueHandler = Environment.i.world.sceneController;
+            BuildDefaultServer(RPCGlobalContext.context);
+        }
+
+        public static RpcServer<RPCContext> BuildDefaultServer(RPCContext context)
+        {
+#if UNITY_WEBGL && !UNITY_EDITOR
+            var transport = new WebGLTransport();
+#else
+            var transport = new WebSocketTransport();
+#endif
+            return BuildDefaultServer(context, transport);
+        }
+
+        public static RpcServer<RPCContext> BuildDefaultServer(RPCContext context, ITransport transport)
+        {
+            return BuildServer(context, transport)
+                   .RegisterService(CRDTServiceImpl.RegisterService)
+                   .Build();
+        }
+
+        public static RPCServerBuilder<RPCContext> BuildServer(RPCContext context, ITransport transport)
+        {
+            return new RPCServerBuilder<RPCContext>(transport, context);
+        }
+    }
+
+    public class RPCServerBuilder<T>
```

Comment:
Don't we need to narrow `T` into `RPCContext` or derived?

Reply:
Didn't add the constrain there because that class don't need it, it doesn't do any data handling, `T` might be whatever you want.
Maybe for some tests we would want to use other context type instead of `RPCContext`, so we have that flexibility by not adding that constrain